src/bscPlugin/CallExpressionInfo.ts

```diff
@@ -66,11 +66,11 @@ export class CallExpressionInfo {
             this.newExpression = callExpression.parent;
         }
         if (isCallfuncExpression(callExpression)) {
-            this.name = callExpression.methodName.text;
+            this.name = (callExpression as CallfuncExpression).methodName.text;
```

Comment:
What is happening here that we need this `as`?

Reply:
I think this was introduced because of the reflection changes. actual types were updated to resolve this